bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/creators/ValueCreator.java

```diff
@@ -608,7 +607,7 @@ public static BXml createXmlProcessingInstruction(BString target, BString data,
      * @param keyValues initial map values to be populated.
      * @return map value
      */
-    public static BMap<BString, Object> createMapValue(MapType mapType, BMapInitialValueEntry[] keyValues) {
+    public static BMap<BString, Object> createMapValue(Type mapType, BMapInitialValueEntry[] keyValues) {
```

Comment:
can we keep using `MapType`

Reply:
This had to be changed because though we can use a `RecordType` to create a `MapValue`, we cannot cast a `RecordType` to a `MapType`.

Reply:
The similar format is followed in the overloaded function prior to this in line 599/600